desktop/apps/auction2/components/basic_checkbox/index.js

```diff
@@ -0,0 +1,24 @@
+import { default as React, PropTypes } from 'react';
+
+export default function BasicCheckbox({ item, onClickAction, checked, onClick }) {
+  return (
+    <div className='auction2-basic-checkbox'>
+      <div className='artsy-checkbox' onClick={() => onClick(item.id)}>
+        <div className='artsy-checkbox--checkbox'>
+          <input
+            type='checkbox'
+            name='artistAggregation'
+            value={item.id}
+            checked={checked}
+            onChange={() => {}} // necessary to get around warning
```

Comment:
Ah, sorry I misrepresented the warning-- it's actually about the `checked` param. But since we have hooked up our own handling to this component (which I want to reuse), the only way to get around the warning would be to set `defaultChecked` (similar to what you linked to above), but then the component won't update as we expect.

This is just a non-standard use of the react checkbox component. I could get rid of the no-op, but the warning is irrelevant. There are some issues open in react about having a better way to suppress these kinds of warnings, but I think for now it's fine.